libs/server/ACL/ACLParser.cs

```diff
@@ -179,11 +179,11 @@ public static void ApplyACLOpToUser(ref User user, string op)
                     user.AddCategory(category);
                 }
             }
-            else if ((op == "~*") || (op == "allkeys"))
+            else if (op.Equals("~*", StringComparison.OrdinalIgnoreCase) || op.Equals("allkeys", StringComparison.OrdinalIgnoreCase))
```

Comment:
No case insestivity needed.
```suggestion
            else if (op.Equals("~*", StringComparison.Ordinal) || op.Equals("allkeys", StringComparison.OrdinalIgnoreCase))
```